tekton/README.md

```diff
@@ -16,110 +16,7 @@ that we use.
-1. [Run the Pipeline](#run-the-pipeline)
-1. Create the new tag and release in GitHub
-   ([see one of way of doing that here](https://github.com/tektoncd/plumbing/tree/master/tekton/resources/release#create-draft-release)).
-1. Add an entry to [the README](../README.md) at `HEAD` for docs and examples for
-   the new release ([README.md#read-the-docs](../README.md#read-the-docs)).
-1. Update [the catalog repo](https://github.com/tektoncd/catalog) test infrastructure to use the
-   new release by updating the `RELEASE_YAML` link in
-   [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/v1beta1/test/e2e-tests.sh).
-
-### Run the Pipeline
-
-To use [`tkn`](https://github.com/tektoncd/cli) to run the `publish-tekton-pipelines` `Task` and create a release:
-
-1. Pick the revision you want to release and add a
-   `PipelineResoruce` for it, e.g.:
-
-   ```yaml
-   apiVersion: tekton.dev/v1alpha1
-   kind: PipelineResource
-   metadata:
-     name: tekton-pipelines-vX-Y-Z
-   spec:
-     type: git
-     params:
-     - name: url
-       value: https://github.com/tektoncd/pipeline
-     - name: revision
-       value: revision-for-vX.Y.Z-invalid-tags-boouuhhh # REPLACE with the commit you'd like to build from (not a tag, since that's not created yet)
-   ```
-
-1. To run against your own infrastructure (if you are running
-   [in the production cluster](https://github.com/tektoncd/plumbing#prow) the
-   default account should already have these creds, this is just a bonus - plus
-   `release-right-meow` might already exist in the cluster!), also setup the
-   required credentials for the `release-right-meow` service account, either:
-
-   - For
-     [the GCP service account `release-right-meow@tekton-releases.iam.gserviceaccount.com`](#production-service-account)
-     which has the proper authorization to release the images and yamls in
-     [our `tekton-releases` GCP project](https://github.com/tektoncd/plumbing#prow)
-   - For
-     [your own GCP service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
-     if running against your own infrastructure
-
-
-1. [Connect to the production cluster](https://github.com/tektoncd/plumbing#the-dogfooding-cluster):
-
-    ```bash
-    gcloud container clusters get-credentials dogfooding --zone us-central1-a --project tekton-releases
-    ```
-
-1. Run the `release-pipeline` (assuming you are using the production cluster and
-   [all the Tasks and Pipelines already exist](#setup)):
-
-   ```shell
-   # Create the resources - i.e. set the revision that you wan to build from
-   kubectl apply -f tekton/resources.yaml
-
-   # Change the environment variable to the version you would like to use.
-   export TEKTON_VERSION=vX.Y.Z
-   export TEKTON_RELEASE_GIT_RESOURCE=tekton-pipelines-git-vX-Y-Z
-   export TEKTON_BUCKET_RESOURCE=pipeline-tekton-bucket
-   export IMAGE_REGISTRY=gcr.io/tekton-releases
```

Comment:
i notice the format of the env vars in the cheat sheet is a bit different from these - i like keeping them in line with the env vars in the example at https://github.com/tektoncd/plumbing/tree/master/tekton/resources/release#create-draft-release

so i can use the same env vars for both

basically what i end up running is something like:

```bash
export TEKTON_VERSION=v0.16.3
export TEKTON_RELEASE_GIT_RESOURCE=tekton-pipelines-git

# I like that you just hardcode these 2
export TEKTON_BUCKET_RESOURCE=pipeline-tekton-bucket
export IMAGE_REGISTRY=gcr.io/tekton-releases

tkn --context dogfood pipeline start \
		--param=versionTag=${TEKTON_VERSION} \
		--param=imageRegistry=${IMAGE_REGISTRY} \
		--serviceaccount=release-right-meow \
		--resource=source-repo=${TEKTON_RELEASE_GIT_RESOURCE} \
		--resource=bucket=${TEKTON_BUCKET_RESOURCE} \
		--resource=builtBaseImage=base-image \
		--resource=builtEntrypointImage=entrypoint-image \
		--resource=builtKubeconfigWriterImage=kubeconfigwriter-image \
		--resource=builtCredsInitImage=creds-init-image \
		--resource=builtGitInitImage=git-init-image \
		--resource=builtControllerImage=controller-image \
		--resource=builtWebhookImage=webhook-image \
		--resource=builtDigestExporterImage=digest-exporter-image \
		--resource=builtPullRequestInitImage=pull-request-init-image \
		--resource=builtGcsFetcherImage=gcs-fetcher-image \
		--resource=notification=post-release-trigger \
                       --resource=builtNopImage=nop-image \
		pipeline-release

export TEKTON_PACKAGE=tektoncd/pipeline
export TEKTON_OLD_VERSION=v0.16.2
export TEKTON_RELEASE_NAME="Dolores Rex"

tkn --context dogfood task start \
  -i source=$TEKTON_RELEASE_GIT_RESOURCE \.  # using the same env var i used in the release pipeline
  -i release-bucket=$TEKTON_BUCKET_RESOURCE \
  -p package=$TEKTON_PACKAGE \
  -p release-tag=$TEKTON_VERSION \
  -p previous-release-tag=$TEKTON_OLD_VERSION \
  -p release-name=$TEKTON_RELEASE_NAME \
  create-draft-release


```

Reply:
Thanks for this, much simpler and better not to create intermediate TaskRun file I think.

I've updated with these changes.